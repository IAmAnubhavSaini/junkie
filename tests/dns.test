#!/bin/sh

NAME="DNS"
PCAPS="pcap/dns/dns.pcap pcap/dns/llmnr.pcap pcap/dns/nbns.pcap pcap/dns/mdns.pcap pcap/dns/ptr_code.pcap"
filter() {
	grep '^DNS' | cut -d' ' -f 2-
}
OUTPUT="dns.out"

. "$srcdir/dump_test"
