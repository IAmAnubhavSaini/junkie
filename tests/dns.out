-- pcap/dns/dns.pcap --
head_len=32, payload=0, QUERY, tx_id=3804, err_code=0, request_type=A, dns_class=IN, name=www.google.com
head_len=32, payload=0, QUERY, tx_id=51952, err_code=0, request_type=AAAA, dns_class=IN, name=www.google.com
head_len=284, payload=0, ANSWER, tx_id=3804, err_code=0, request_type=A, dns_class=IN, name=www.google.com
-- pcap/dns/llmnr.pcap --
head_len=24, payload=0, QUERY, tx_id=1304, err_code=0, request_type=ANY, dns_class=IN, name=stpbi4
-- pcap/dns/nbns.pcap --
head_len=50, payload=0, QUERY, tx_id=43573, err_code=0, request_type=NB, dns_class=IN, name=UMTOURNAISIS.BE
-- pcap/dns/mdns.pcap --
head_len=45, payload=0, QUERY, tx_id=0, err_code=0, request_type=PTR, dns_class=IN, name=225.80.168.192.in-addr.arpa
-- pcap/dns/ptr_code.pcap --
head_len=92, payload=0, QUERY, tx_id=0, err_code=0, request_type=SRV, dns_class=IN, name=SFTP File Transfer on optiplexquad._sftp-ssh._tcp.local
head_len=116, payload=0, ANSWER, tx_id=0, err_code=0, request_type=UNSET, dns_class=UNSET, name=
